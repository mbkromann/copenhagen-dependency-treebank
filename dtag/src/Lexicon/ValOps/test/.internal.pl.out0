ValOp([a]-[b], 5)
hash([a=b,c=d]-[c], 1)
list([a,b]-[a], 2)
set([a,b]-[a], 2)
