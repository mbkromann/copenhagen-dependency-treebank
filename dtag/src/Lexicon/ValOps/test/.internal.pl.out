ERROR! cannot read file /opt/dta/arcs.header in Graph->readfile
check that DTAGHOME is set correctly!
ERROR! cannot read file /opt/dta/arcs.trailer in Graph->readfile
check that DTAGHOME is set correctly!
ValOp([a]-[b], 5)
hash([a=b,c=d]-[c], 1)
list([a,b]-[a], 2)
set([a,b]-[a], 2)
